database/migrations/2016_10_20_071655_added_by_user_id_column_to_payments_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddedByUserIdColumnToPaymentsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('payments', function (Blueprint $table) {
+            $table->integer('addedByUserId')->nullable();
```

Comment:
nonstop mi na tom padajú migrácie, doplň to tam ak vieš ako a treba to tam
$table->integer('addedByUserId')->unsigned();
$table->foreign('addedByUserId')->references('id')->on('users');